tempesta_fw/t/functional/selftests/test_headercollection.py

```diff
@@ -101,13 +101,22 @@ def test_is_equal(self):
 
         reorderd = deproxy.HeaderCollection()
         reorderd.add('C', 'zxcv')
+        reorderd.add('A', 'qwerty')
         reorderd.add('A', 'uiop')
         reorderd.add('A', 'jkl;')
-        reorderd.add('A', 'qwerty')
         reorderd.add('B', 'asdf')
         self.assertTrue(self.headers == reorderd)
         self.assertFalse(self.headers != reorderd)
 
+        same_keys_reorderd = deproxy.HeaderCollection()
+        same_keys_reorderd.add('C', 'zxcv')
+        same_keys_reorderd.add('A', 'uiop')
+        same_keys_reorderd.add('A', 'jkl;')
+        same_keys_reorderd.add('A', 'qwerty')
+        same_keys_reorderd.add('B', 'asdf')
+        self.assertTrue(self.headers != same_keys_reorderd)
+        self.assertFalse(self.headers == same_keys_reorderd)
```

Comment:
Please also add test, where some singular header will be repeated twice. That header collection must not be equal to original one.